staging/src/k8s.io/client-go/kubernetes/typed/core/v1/pod_expansion.go

```diff
@@ -21,24 +21,25 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	policy "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes/scheme"
 	restclient "k8s.io/client-go/rest"
 )
 
 // The PodExpansion interface allows manually adding extra methods to the PodInterface.
 type PodExpansion interface {
-	Bind(binding *v1.Binding) error
-	Evict(eviction *policy.Eviction) error
+	Bind(ctx context.Context, binding *v1.Binding, opts metav1.CreateOptions) error
+	Evict(ctx context.Context, eviction *policy.Eviction) error
```

Comment:
Should this accept options of some sort? `Bind` performs a `Post` and accepts `CreateOptions`. `Evict` _also_ does a `Post`. How come this one is special?

Reply:
DeleteOptions are inside the eviction object, which is sent in the body